### PR TITLE
css-pseudo-classes - James Yang

### PR DESCRIPTION
**Pull request by jamesdae**
https://user-images.githubusercontent.com/110082070/195428725-661d7a51-3524-4272-984c-c523a69f11a9.mov

### CODE REVIEW

File: css-pseudo-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+</head>
+<body>
+  <div>
+    <h3>Hover Button</h3>
+    <button type="button">I'm a button!</button>
+  </div>
+  <div>
+    <h3>Striped Table</h3>
+    <table>
+      <thead>
```
Comment (Zhiwutian) [inline css-pseudo-classes/index.html:15]: the thead element should only cover the `th` elements, the rest should be in the `tbody` elements

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed!

**Comment by jamesdae** [comment]
https://user-images.githubusercontent.com/110082070/195430116-fb7b4ca9-01d6-4035-bbcd-7f839bcc0121.mov

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**